src/components/standfirst.tsx

```diff
@@ -121,7 +121,7 @@ function content(standfirst: DocumentFragment, item: Item): ReactNode {
 
 const Standfirst: FC<Props> = ({ item }) =>
     item.standfirst.fmap<ReactElement | null>(standfirst =>
-        <div css={getStyles(item)}>{content(standfirst, item)}</div>,
+        <div className="standfirst" css={getStyles(item)}>{content(standfirst, item)}</div>,
```

Comment:
What do full-page interactives use this class for?

Reply:
An example I came across was making the standfirst text white here https://www.theguardian.com/society/ng-interactive/2020/apr/29/how-humans-have-reacted-to-pandemics-through-history-a-visual-guide

This is apps-specific for backwards compatibility so hopefully this won't be needed.